tests/dtm/test_single_process_restart.py

```diff
@@ -69,6 +78,24 @@ def setup_class(cls):
         cls.log.info("Setup S3bench")
         resp = s3bench.setup_s3bench()
         assert_utils.assert_true(resp)
+        cls.ha_obj = HAK8s()
+        cls.rest_obj = S3AccountOperations()
+
+    def setup_method(self):
+        """Setup Method"""
+        self.log.info("Check the overall status of the cluster.")
+        resp = self.ha_obj.check_cluster_status(self.master_node_list[0])
+        if not resp[0]:
```

Comment:
@priyankaborawake for health check actually it would fail in start itself as part of framework itself so was just thinking if we really need this but then in case someone wants to disable health check on framework level, we can just check here if any issues due to some intermittent pod failure, it would try to stabilize system with restarting it. But again for this suite, if we restart it might create some other issue and our test might fail so we can assert instead of restart. cc: @ApurwaMohite 
(sorry for long comment but wanted to check on all conditions)

Reply:
@ApurwaMohite Lets assert instead of restarting the cluster.